pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1992,19 +1994,24 @@ public Map<String, Map<String, String>> getAllJobsForTable(String tableNameWithT
     for (ControllerJobType jobType : jobTypes) {
       String jobsResourcePath = ZKMetadataProvider.constructPropertyStorePathForControllerJob(jobType);
       try {
+        if (!_propertyStore.exists(jobsResourcePath, -1)) {
```

Comment:
(minor) Use `AccessOption.PERSISTENT`, same for `get()`

Reply:
This check is actually redundant. The null check itself is good enough

Reply:
as tested, propertyStore.get() threw ZkNoNodeException. The current try-catch would continue the for-loop upon this exception, but the warn logs were kinda annoying, so I added the explicit existence check.